LibraryApp/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { ImageBackground, Image, Text } from 'react-native';
+import PropTypes from 'prop-types';
+import CustomButton from '@components/CustomButton';
+import CustomTextInput from '@components/CustomTextInput';
+import backgroundImg from '@assets/bc_inicio.png';
+import logo from '@assets/Group.png';
+
+import styles from './styles';
+import {
+  EMAIL,
+  EMAIL_PLACEHOLDER,
+  PASSWORD,
+  PASSWORD_PLACEHOLDER,
+  SUBMIT_TEXT,
+  FOOTER_TEXT
+} from './constants';
+
+class Login extends Component {
+  render() {
+    const { onSubmit, onPasswordChange, onEmailChange, emailError, passwordError } = this.props;
+    return (
+      <ImageBackground source={backgroundImg} style={styles.container}>
+        <Image source={logo} style={styles.logo} />
+        <CustomTextInput
+          title={EMAIL}
+          placeholder={EMAIL_PLACEHOLDER}
+          viewStyles={styles.inputStyle}
+          titleStyles={styles.titleStyle}
+          inputTextStyles={styles.inputTextStyle}
+          onChange={onEmailChange}
+        />
+        {emailError && <Text style={styles.errorText}>{emailError}</Text>}
```

Comment:
When we have this kind of repeated behaviour, we create a component `CustomInput` and add the error handling there